### PR TITLE
D8 core 1816 wysiwyg tweaks

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW
- 
# Summary
- This is a bunch of spacing tweaks for the WYSIYWG

# Needed By (Date)
- Code freeze

# Urgency
- Not

# Steps to Test

1. Pull this branch
2. Clear cache
3. Verify:

> 1. UL & OR
> margin-bottom: modular spacer 2
> 
> 2. .su-callout-text
> margin-bottom: modular spacer 3
> 
> 3. h2
> padding-top: modular scale 2
> 
> 4. h3, h4
> margin-bottom: modular scale -1
> 
> 5. intro
> margin-bottom: modular scale 2
> 
> 6. th, table td
> line-height: 1.2em;

# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1816
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/stanford_profile_styles/dist/css/stanford_profile_styles.admin.node_form.css [hunk 1]
```diff
@@ -1 +1,37 @@
-.node-form .entity-meta details[open]{background-image:none}.node-form .entity-meta,.node-form .entity-meta__header,.node-form .entity-meta__header details,.node-form .entity-meta details{border-top:0 transparent;border-bottom:0 transparent}.node-form .entity-meta:first-child,.node-form .entity-meta__header:first-child,.node-form .entity-meta__header details:first-child,.node-form .entity-meta details:first-child{border-top-color:transparent}@media screen and (min-width:768px){.node-form .layout-region-node-footer,.node-form .layout-region-node-main{float:inherit;width:100%;padding-right:0;-webkit-box-sizing:border-box;box-sizing:border-box}.node-form [dir=rtl] .layout-region-node-footer,.node-form [dir=rtl] .layout-region-node-main{float:inherit;padding-left:0;padding-right:0}.node-form .layout-region-node-secondary{float:inherit;width:100%}.node-form [dir=rtl] .layout-region-node-secondary{float:inherit}}
\ No newline at end of file
+.node-form .entity-meta details[open] {
```
Comment (sherakama) [inline modules/stanford_profile_styles/dist/css/stanford_profile_styles.admin.node_form.css:1]: Please make sure you submit with `npm run publish` so this css is minimized.

File: modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_table.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@charset "UTF-8";
+
+.su-wysiwyg-text {
+  table {
+    th,
+    td {
+      line-height: 1.2em;
```
Comment (sherakama) [inline modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_table.scss:7]: Should this be `rems` instead of `ems`?

**Comment by sherakama** [comment]
@cjwest this looks good. Please submit your pull requests after running `npm run publish` and please validate that the unit of `em` is correct over `rem`

**Comment by cjwest** [comment]
Thank you @sherakama !
Ready for your 👀

**Pull request merged**